src/views/components/radios/index.njk

```diff
@@ -0,0 +1,56 @@
+---
+title: Radios
+description: Radios let the user select a single option from a list.
+layout: default
+eleventyNavigation:
+  key: Radios
+  section: components
+---
+{% from "example.njk" import example %}
+
+<section>
+  <p>Radios let the user select a single option from a list.</p>
+  {{ example({group: "components", item: "radios", example: "default", html: true, open: false }) }}
+</section>
+
+<section class="my-8">
+  <h2 class="mb-4">When to use this component</h2>
+  <p>Use the radio component when you need to help users select a single option from a list.</p>
+</section>
+
+<section class="mb-8">
+  <h2 class="mb-4">When not to use this component</h2>
+  <p>Do not use the radios component if users can choose more than one option from a list. In this case, use the <a href="/components/checkbox">checkboxes</a> component.</p>
+</section>
+
+<section class="mb-8">
+  <h2 class="mb-4">How to use this component</h2>
+  <p>Radios are grouped within a <code>fieldset</code> which starts with a <code>legend</code> which lets the user know what they need to do.</p>
+</section>
+
+<section class="mb-8">
+  <h2 class="mb-4">Variants</h2>
+
+  <h3 class="mb-4">Descriptions</h3>
+  <p>This variant allows you to add more context to the options in the list. The description is populated by providing a <code>description</code> parameter within the radio’s <code>label</code> parameter.</p>
+  {{ example({group: "components", item: "radios", example: "with-descriptions", html: true, open: false }) }}
+
+  <h3 class="mb-4">Other</h3>
+  <p>This variant lets the user provide an answer other to the options in the list, or to provide supplemental information required to describe the nature of their answer.</p>
+  <p>The radio label must be directly followed by the <code>radio__description</code> to let the user know they will need to provide an answer if they select this option.</p>
+  <p>Use this variation by wrapping the text field in a div with class <code>radio__other</code>, added directly after the radio label. CSS hides this field until the radio is <code>:checked</code>.</p>
+  {{ example({group: "components", item: "radios", example: "other", html: true, open: false }) }}
+
+  <h3 class="mb-4">Other - visible input</h3>
+  <p>This variant of the ‘Other’ radio input type shows the input instead of waiting for the radio to be <code>:checked</code>. This variant does not require the <code>radio__description</code> to be present.</p>
+  {{ example({group: "components", item: "radios", example: "other--visible-input", html: true, open: false }) }}
+
+  <h3 class="mb-4">Other - select</h3>
+  <p>This variant of the ‘Other’ radio input type uses a select instead of an input. Select “pepperoni” for select to become visible.</p>
+  {{ example({group: "components", item: "radios", example: "other--select", html: true, open: false }) }}
```

Comment:
In the `Select` page there is a warning:

> These should never be used for short lists of items that could be better served by using radio options.

Consider adding something similar here, unless it does not apply